Client/mods/deathmatch/logic/luadefs/CLuaGUIDefs.cpp

```diff
@@ -2945,6 +2947,28 @@ int CLuaGUIDefs::GUIEditSetReadOnly(lua_State* luaVM)
     return 1;
 }
 
+int CLuaGUIDefs::GUIEditIsReadOnly(lua_State* luaVM)
+{
+    // bool guiEditIsReadOnly( element editField )
+    CClientGUIElement* editField;
+
+    CScriptArgReader argStream(luaVM);
+    argStream.ReadUserData<CGUIEdit>(editField);
```

Comment:
You don't need to and shouldn't write `<CGUIEdit>` here.

Just `argStream.ReadUserData(editField)` is perfectly fine.